schemas/CreateAccountInput.json

```
@@ -0,0 +1,271 @@
+                },
+                "repay_name_on_check": {
+                  "type": "string",
+                  "description": "Account holder's name as it appears on the account.",
+                  "example": "Lucille Bluth"
+                }
+              }
+            }
+          }
+        },
+        "cycle_type": {
+          "type": "object",
+          "properties": {
+            "first_cycle_interval": {
+              "type": "string",
+              "nullable": true,
+              "description": "Interval for a first cycle for this account.",
+              "example": "11 days"
+            }
+          }
+        },
+        "summary": {
+          "type": "object",
+          "properties": {
+            "credit_limit_cents": {
+              "type": "integer",
+              "description": "Total Amount (in cents) that this account can borrow.",
+              "example": 400000
+            },
+            "max_approved_credit_limit_cents": {
+              "type": "integer",
+              "description": "Total Amount (in cents) that this account can borrow.",
+              "example": 400000
+            },
+            "late_fee_cents": {
+              "type": "integer",
+              "description": "The fee charged for late payments on the account.",
+              "example": 2900
+            },
+            "payment_reversal_fee_cents": {
+              "type": "integer",
+              "description": "The fee charged for payment reversals on the account.",
+              "example": 2900
+            },
+            "origination_fee_cents": {
+              "type": "integer",
+              "description": "The fee charged at the time of account origination",
+              "default": 0,
+              "example": 2000
+            },
+            "annual_fee_cents": {
+              "type": "integer",
+              "description": "An annual fee to be charged yearly from the date of account creation. It will reflect on the subsequent statement once incurred.",
+              "default": 0,
+              "example": 1000
+            },
+            "monthly_fee_cents": {
+              "type": "integer",
+              "description": "A monthly fee to be charged monthly from the date of account creation. It will reflect on the subsequent statement once incurred.",
+              "default": 0,
+              "example": 500
+            },
+            "initial_principal_cents": {
+              "type": "integer",
+              "description": "The initial principal balance (in cents) for the account.",
+              "example": 400000,
+              "nullable": true
+            }
+          }
+        },
+        "promo_overview": {
+          "type": "object",
+          "properties": {
+            "promo_impl_interest_rate_percent": {
+              "type": "number",
+              "description": "The percentage interest applied to the account during the promotional period (i.e. 6.2%)",
```

eg for examples, ie for alternate explanations
```suggestion
              "description": "The percentage interest applied to the account during the promotional period (e.g. 6.2%)",
```